pkg/kubelet/stats/host_stats_provider.go

```diff
@@ -79,6 +80,11 @@ func (h hostStatsProvider) getPodContainerLogStats(podNamespace, podName string,
 
 // getPodEtcHostsStats gets status for pod etc hosts usage
 func (h hostStatsProvider) getPodEtcHostsStats(podUID types.UID, rootFsInfo *cadvisorapiv2.FsInfo) (*statsapi.FsStats, error) {
+	if runtime.GOOS == "windows" {
+		// /etc/hosts file is not mounted for windows
```

Comment:
This reminded me that we should probably get https://github.com/kubernetes/kubernetes/pull/83730 merged.
That PR adds support for /etc/hosts mapping for non dockershim configurations on Windows.

Reply:
Is there a way to determine if that functionality is enabled and we should add an additional check here?

Reply:
83730 added a method to check if the node supports /etc/hosts mapping (`supportsSingleFileMapping := kl.containerRuntime.SupportsSingleFileMapping()`) you could use the same function here.

Reply:
I've updated this to included support for containerd on windows.